bindings_ffi/src/mls.rs

```diff
@@ -167,18 +167,35 @@ pub struct FfiConversations {
     inner_client: Arc<RustXmtpClient>,
 }
 
+#[derive(uniffi::Enum)]
+pub enum GroupPermissions {
+    EveryoneIsAdmin,
+    GroupCreatorIsAdmin,
+}
+
 #[uniffi::export(async_runtime = "tokio")]
 impl FfiConversations {
     pub async fn create_group(
         &self,
         account_addresses: Vec<String>,
+        permissions: Option<GroupPermissions>,
     ) -> Result<Arc<FfiGroup>, GenericError> {
         log::info!(
             "creating group with account addresses: {}",
             account_addresses.join(", ")
         );
 
-        let convo = self.inner_client.create_group()?;
+        let group_permissions = match permissions {
+            Some(GroupPermissions::EveryoneIsAdmin) => {
+                Some(xmtp_mls::groups::PreconfiguredPolicies::EveryoneIsAdmin)
+            }
+            Some(GroupPermissions::GroupCreatorIsAdmin) => {
+                Some(xmtp_mls::groups::PreconfiguredPolicies::GroupCreatorIsAdmin)
+            }
+            _ => None,
```

Comment:
Should we default this to `EveryoneIsAdmin`? Or is it intended to mean nobody is an admin here? If so, should we add that as an explicit option rather than the default behavior?

Reply:
Ya. It's a little confusing. `None` in practice becomes `EverybodyIsAdmin` because we have that implemented using the `Default` trait, which gets used when this passes back None.